src/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -89,6 +95,15 @@ export const AuctionsAppFragmentContainer = createFragmentContainer(
     viewer: graphql`
       fragment AuctionsApp_viewer on Viewer {
         ...CuritorialRailsTabBar_viewer
+        me {
```

Comment:
Note that this is a very (very) slow query, so we should avoid adding this to the page load time just so we can emit a visual spacer. 

A better approach would be to update `MyBidsQueryRenderer` with the spacer. The query renderer loads on the client, and does not block rendering during the SSR pass.

Reply:
Yes, thought about it but [here](https://github.com/artsy/force/blob/f13712879fce044acafcf45cb16ea132db120311/src/Apps/Home/HomeApp.tsx#L51) on the Homepage we're wrapping all of the Rails in `<Join separator={<Spacer y={[6, 12]} />}>` so in that case we would also have to rework that as well.

Reply:
In that case, can do something like: 

```tsx
MyBidsQueryRenderer mt={12} />
```
and on the inside:

```tsx
interface MyBidsQueryRendererProps extends BoxProps {}

const MyBidsMyBidsQueryRenderer: React.FC<MyBidsQueryRendererProps> = ({
  mt = 0, // default arg for home 
  ...spaceProps
}) => {
  return (
    <Box mt={mt} {...spaceProps}>
      <SystemQueryRenderer ... />
    </Box>
  )
}

Reply:
Yes, now that you mentioned that it's very slow, would be good to avoid that query especially in this case. Will rework this!